cores/migrations/20240725022254_MakeFetchedAtNonNull.up.sql

```diff
@@ -0,0 +1,5 @@
+-- Add migration script here
+ALTER TABLE page_snapshots
+ALTER COLUMN fetched_at
+SET
+  NOT NULL;
```

Comment:
We save a page_snapshots when we have fetched the html so there isn't a chance for this to be null